helm/charts/hydra/values.yaml

```diff
@@ -147,3 +147,15 @@ affinity: {}
 # Configures controller setup
 maester:
   enabled: true
+
+# Values for the hydra-maester chart
+hydra-maester:
```

Comment:
Would it be possible to move this under `master` and `config` or does this have to reflect the subchart name?

Reply:
It does not have to match the chart name directly, but it does have to be contained in a single object. It could be renamed to `maester` and `maester.enabled` can still be checked but the subdoc has to be together as one